markdown/blog/2024-december-and-paris.md

```diff
@@ -0,0 +1,138 @@
+
+The AsyncAPI Conf track took place on the 3rd day of the conference, featuring an impressive lineup of sessions that attracted a diverse audience. The event was consistently packed, with attendees engaged throughout the day.
+
+<Figure
+  src="/img/posts/paris-conf-2024/full-room.webp"
+  caption="Attendees engaged through the sessions."
+  className="text-center"
+/>
+
+Fran and Lukasz started the track with a welcome speech and mentioned that AsyncAPI recently celebrated its 8th anniversary in November.
+
+<Figure
+  src="/img/posts/paris-conf-2024/lukasz-and-fran.webp"
+  caption="Fran and Lukasz during the welcome speech."
+  className="text-center"
+/>
+
+Naresh Jain and Pierre Gauthier presented their keynote on `TM Forum's AsyncAPI for a New Era of Event-Driven Architecture`. During the session, they announced AsyncAPI as the standard, with over 100 telecommunications APIs already in production. TM Forum has around 800 telco companies, and they will implement all APIs in an async manner, extensively utilizing the request-reply pattern.
+
+<Figure
+  src="/img/posts/paris-conf-2024/naresh-and-pierre.webp"
+  caption="Naresh and Pierre presenting their keynote."
+  className="text-center"
+/>
+
+Frank demonstrated how treating API governance as an enabler unlocks the ability to deliver compelling developer experiences for producers and consumers in event-driven architecture (EDA).
+
+<Figure
+  src="/img/posts/paris-conf-2024/frank.webp"
+  caption="Frank presenting on API Governance for EDA: Unlocking Developer Experience with AsyncAPI."
+  className="text-center"
+/>
+
+Leonid shared how EDAs introduce new data models, protocols, and APIs not found in the traditional REST/CRUD application stack. And how one can abstract these elements to make them feel familiar to application developers, allowing them to create streaming applications seamlessly.
+
+<Figure
+  src="/img/posts/paris-conf-2024/leonid.webp"
+  caption="Leonid presenting on Reimagining Streaming Apps with AsyncAPI and Postgres."
+  className="text-center"
+/>
+
+Hugo then shared how the AsyncAPI Initiative is not only in charge of the specification but has created open-source projects to make it easier for developers to work with the specification documents.
+
+<Figure
+  src="/img/posts/paris-conf-2024/hugo.webp"
+  caption="Hugo presenting on Using the AsyncAPI Ecosystem for Your Event-Driven Architecture."
+  className="text-center"
+/>
+
+Julien and Alessandro shared how organizations can harness the full potential of event-driven integration by leveraging GoldenGate Data Streams, AsyncAPI, and Solace PubSub+ Event Mesh.
```

Comment:
```suggestion
Julien and Alessandro shared how organizations can harness the full potential of event-driven integration by leveraging GoldenGate Data Streams, AsyncAPI, and Solace PubSub+ Event Mesh. They shared a great use case for AI and how it can be trained real-time and standardized with AsyncAPI.
```